components/Investment/TrancheMetric/index.tsx

```diff
@@ -1,48 +1,63 @@
 import * as React from 'react';
 import { Box, Heading } from 'grommet';
-import { baseToDisplay, Investor, Tranche } from 'tinlake';
-import NumberDisplay from '../../NumberDisplay';
+import { Investor, Tranche } from 'tinlake';
 import { TrancheType } from '../../../services/tinlake/actions';
 import DashboardMetric from '../../DashboardMetric';
+import { Erc20Widget } from '../../../components/erc20-widget';
+import DAI from '../../../static/dai.json';
 
 interface Props {
   investor: Investor;
   tranche: Tranche;
   type: TrancheType;
+  tinlake: any;
 }
 
 class TrancheMetric extends React.Component<Props> {
+
   render() {
     const { type, investor, tranche } = this.props;
     const { maxSupply, maxRedeem, tokenBalance } = investor[type];
     const { token  } = tranche;
-    const currencyLabel = ` ${token}`;
-
+    const dropAddress = this.props.tinlake.contractAddresses.SENIOR_TOKEN as string;
+    const tinAddress = this.props.tinlake.contractAddresses.JUNIOR_TOKEN as string;
+    const dropToken = {
+      [dropAddress] : {
+        symbol: 'DROP',
+        logo: '../../static/DROP_final.svg',
+        decimals: 18,
+        name: 'DROP'
+      }
+    };
+    const tinToken = {
+      [tinAddress] : {
+        symbol: 'TIN',
+        logo: '../../static/TIN_final.svg',
+        decimals: 18,
+        name: 'TIN'
+      }
+    };
```

Comment:
I think the InvestorView Container should take care of retrieving all this informations and making the distinction between junior & senior infos based on the tab selected.
you can extend the tranche object by all the required values and pass it down to this component from the investorView container.
So, this component will just display all the infos that it gets without even bothering if it is dealing with junior or senior data. 

If you get any type issues when extending the  tranche object, please just use any. I will fix it after the pr is merged.


Then you also would not need to add tinlake to the props.